README.md

```diff
@@ -8,18 +8,109 @@
 
 Streaming Jupyter Integrations project includes a set of magics for interactively running _Flink SQL_  jobs in [Jupyter](https://jupyter.org/) Notebooks
 
+## Installation
+
 In order to actually use these magics, you must install our PIP package along `jupyterlab-lsp`:
 
 ```shell
 python3 -m pip install jupyterlab-lsp streaming-jupyter-integrations
 ```
 
-And then register in Jupyter with a running IPython in the first cell:
+## Usage
+
+Register in Jupyter with a running IPython in the first cell:
 
 ```python
 %load_ext streaming_jupyter_integrations.magics
 ```
 
+Then you need to decide which _execution mode_ and _execution target_ to choose.
+
+```python
+%flink_connect --execution-mode [mode] --execution-target [target]
+```
+
+By default, the streaming execution mode and local execution target are used.
```

Comment:
maybe emphasize mode and target with `this` so they are more visible

Reply:
done